### DISCUSSION TITLE
How to create 1 observer 'total fraction of dose' for multiple molecules/enzymes

### DISCUSSION THREAD

**Discussion by nina-nauwelaerts** [Open-Systems-Pharmacology/Forum#923] [How-Tos] [answered]
Hi 

I am working on a compound that is metabolized via multiple routes, including different UGT enzymes. The container observer 'Total fraction of dose' gives me the fraction for each of the enzymes. However, I would like to have a single observer that gives immediately the total fraction for all UGT enzymes. 
I am aware of the examples in the manual, but I'm still not able to create this observer.

Thanks!

**Comment by Yuri05** [comment, marked as answer]
One possibility to do this is:
* Caclulate the total amount for each metabolite
* Define the observer as `(Total_amount_metabolite_1+...+Total_amount_metabolite_N) / Total_DrugMass_of_the_parent_compound`

E.g. see the attached example. [TotalFractionDose_OverAllMetabolites_02.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/6696065/TotalFractionDose_OverAllMetabolites_02.zip)
Here the Compound **C1** is metabolized by 2 Enzymes (**Enzyme_1** and **Enzyme_2**) building 2 metabolites: **C1-Enzyme_1-aa Metabolite** and **C1-Enzyme_2-bb Metabolite**
1. In the molecules building block, I introduce the Parameter "**Total drug amount in organsim**" for the first metabolite and define is as the sum of all amounts of this metabolite within the organism (important: set parameter type to Property)
![grafik](https://user-images.githubusercontent.com/25061876/122965965-1159a500-d389-11eb-82e0-63ff94589130.png)
2. Same for the 2nd metabolite (here I can just use the same formula which I created for the first metabolite)
![grafik](https://user-images.githubusercontent.com/25061876/122966188-4fef5f80-d389-11eb-92a4-1a40f410f8b9.png)
3. Now I define new **container** observer for the total fraction of all metabolites
![grafik](https://user-images.githubusercontent.com/25061876/122966519-b83e4100-d389-11eb-8557-381914f322ba.png)
4. The formula of the new observer is then:
![grafik](https://user-images.githubusercontent.com/25061876/122966721-f0458400-d389-11eb-85cf-bdd334703410.png)
5. Now you can plot the new observer
![grafik](https://user-images.githubusercontent.com/25061876/122967215-85e11380-d38a-11eb-9b9a-8b7d43aa99df.png)
